src/ProtobufServer/ecdar_requests/send_query.rs

```diff
@@ -74,6 +74,11 @@ fn convert_ecdar_result(query_result: &QueryResult) -> Option<ProtobufResult> {
             success: *refines,
             relation: vec![],
         })),
+
+        QueryResult::Reachability(_, _) =>{
+            panic!("Not implemented, but should be implemented");
+        }
+
```

Comment:
We should implement this before merging probably

Reply:
For sprint 3